manifests/a/Agilent/LabAdvisor/2.19.503/Agilent.LabAdvisor.locale.en-US.yaml

```diff
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.0.6 $debug=AUSU.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Agilent.LabAdvisor
+PackageVersion: 2.19.503
+PackageLocale: en-US
+Publisher: Agilent Technologies
+# PublisherUrl: 
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+Author: Agilent
+PackageName: Agilent Lab Advisor
+PackageUrl: https://www.agilent.com/en/product/liquid-chromatography/hplc-ce-software/instrument-control-software/lab-advisor-software
+License: Copyright (c) Agilent Technologies
+LicenseUrl: https://labadvisor.pl29.agilent.com/Software%20License%20Terms.txt
+# Copyright: 
```

Comment:
```suggestion
License: Proprietary
LicenseUrl: https://labadvisor.pl29.agilent.com/Software%20License%20Terms.txt
Copyright: Copyright (c) Agilent Technologies
```

Reply:
Yes you are right, thanks for the change!